src/poetry_bundle_plugin/bundlers/venv_bundler.py

```diff
@@ -130,7 +130,7 @@ def bundle(self, poetry: "Poetry", io: "IO") -> bool:
             poetry.pool,
             poetry.config,
         )
-        installer.remove_untracked()
+        installer.requires_synchronization(True)
```

Comment:
There should be no need to explicitly pass the default argument here. Otherwise, LGTM.